components/build/.tekton/serviceaccount.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: pipeline
+secrets:
+  - name: redhat-appstudio-staginguser-pull-secret
```

Comment:
I can add this secret to the cluster

Reply:
Could you also check quay robot permission from description? Or even that webhook?